src/Symfony/Component/Security/Acl/Domain/Entry.php

```diff
@@ -178,14 +178,18 @@ public function setStrategy($strategy)
      */
     public function serialize()
     {
-        return serialize(array(
-            $this->mask,
-            $this->id,
-            $this->securityIdentity,
-            $this->strategy,
-            $this->auditFailure,
-            $this->auditSuccess,
-            $this->granting,
+        $args = func_get_args();
```

Comment:
hmm, actually, I see that you are calling it with an argument in the child class. But this is a complete hack